Fix blockquote nested lists formatting.

hokukim opened the pull request:
Fix blockquote nested lists formatting in "DNS domain name" guidelines section. This fix allows GitHub's markdown interpreter to correctly interpret the intended formatting, which is a blockquote containing a list and a nested list.

azurecla commented:
Hi __@srehorn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@srehorn : Thanks for your contribution to the Azure documentation! The author, @btardif, has been notified to review your proposed change.

btardif commented:
Looks good, thanks for the correction! #sign-off

btardif commented:
#sign-off

btardif commented:
@tysonn this can be merged as is.

hokukim committed:
Fix blockquote nested lists formatting.

Fix blockquote nested lists formatting in "DNS domain name" guidelines section. This fix allows GitHub's markdown interpreter to correctly interpret the intended formatting, which is a blockquote containing a list and a nested list.
